clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -250,7 +250,8 @@ Changes in existing checks
 
 - Improved :doc:`cppcoreguidelines-prefer-member-initializer
   <clang-tidy/checks/cppcoreguidelines/prefer-member-initializer>` check to
-  ignore delegate constructors.
+  ignore delegate constructors and ignore re-assignment for reference or after
+  unsafety assignment.
```

Comment:
would be good to clarify what unsafety means here, maybe "ignore re-assignment for reference or when initialization depend on field that is initialized before."